core/sql/common/ComMisc.cpp

```diff
@@ -99,3 +99,17 @@ NABoolean ComIsTrafodionReservedSchemaName(
   return FALSE;
 }
 
+// schema names of pattern "_HV ... _" and "_HB_ ... _" are reserved to store
```

Comment:
sorry, I did not catch the "underscore" comment - I will fix this in a subsequent delivery.